edit/buildozer.go

```diff
@@ -692,8 +700,13 @@ func rewrite(commandsForFile commandsForFile) *rewriteResult {
 		}
 	} else {
 		origName := name
-		name = strings.TrimSuffix(name, "BUILD")
-		for _, suffix := range buildFileSuffixes {
+		for suffix := range buildFileNames {
```

Comment:
the problem with this change is the intentionally random iteration order of Go maps.

You probably need to keep the list for this part and the map for below